msal/src/test/res/raw/aad_capabilities_cp1.json

```diff
@@ -0,0 +1,17 @@
+{
+  "client_id" : "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0",
+  "authorization_user_agent" : "DEFAULT",
+  "redirect_uri" : "msauth://com.microsoft.identity.client.sample.local/1wIqXSqBj7w%2Bh11ZifsnqwgyKrY%3D",
+  "multiple_clouds_supported":true,
+  "broker_redirect_uri_registered": true,
+  "account_mode": "MULTIPLE",
+  "client_capabilities": "CP1",
```

Comment:
nit: remove `comma` we have only one capability.

Reply:
Comma is needed to separate client_capabilities from authorities.  No comma in the value for client_capabilities.